recipes/django-axes/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "django-axes" %}
+{% set version = "5.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c8f1234003536e8eadc624745a2d302f9251943845a8df770c73e2bdb8782632
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - django
+    - black
+    - mypy
```

Comment:
So in the project's `install_requires`, it lists: `"django>=2.0", "django-appconf>=1.0.3", "django-ipware>=3,<4"`, but none of black, mypy, or prospector, which are really developer tools and IMO shouldn't be listed here.

Reply:
Okay, i see the problem. I'm working on the dependencies now.